src/models/Projection.ts

```diff
@@ -277,36 +305,69 @@ export class Projection {
     return result;
   }
 
-  private calcIcuUtilization(
+  private useActualTimeseries(actualTimeseries: ActualsTimeseries) {
+    // make sure there are recent datapoints.
+    if (actualTimeseries.length < 3) return false;
+    let count = 0;
+    for (var i = 0; i < 5; i++) {
+      const actual = actualTimeseries[actualTimeseries.length - 1 - i];
+      if (actual.ICUBeds.currentUsageCovid && actual.ICUBeds.totalCapacity) {
+        count += 1;
+      }
+    }
+    return count > 3;
+  }
+
+  private calcICUHeadroom(
+    actualTimeseries: ActualsTimeseries,
     timeseries: Timeseries,
     lastUpdated: Date,
   ): Array<number | null> {
-    // The API gives us the beds in use *by covid*, and the total capacity *for
-    // covid*, using an assumption that ICUs are usually 75% full with non-covid
-    // patients. We've decided to show full ICU utilization (not just covid), so
-    // we need to undo that assumption.
-    // TODO(igor): Update this on the API side so we can undo this logic.
-    const icuUtilization = timeseries.map(row => {
-      if (row.ICUBedCapacity > 0 && row.ICUBedsInUse > 0) {
-        return Math.min(1, row.ICUBedsInUse / row.ICUBedCapacity);
-      } else {
-        return null;
-      }
-    });
-
+    let icuUtilization;
+
+    if (this.hasActualData) {
+      icuUtilization = actualTimeseries.map(row => {
+        if (row.ICUBeds.currentUsageCovid > 0 && row.ICUBeds.totalCapacity) {
+          // use the actual icu patients total here
+          const actualIcuPatientsToal = row.ICUBeds.currentUsageTotal;
+          const icuHeadroomUsed =
+            row.ICUBeds.currentUsageCovid /
+            (row.ICUBeds.totalCapacity -
+              (actualIcuPatientsToal - row.ICUBeds.currentUsageCovid));
```

Comment:
I don't _think_ we want to subtract `row.ICUBeds.currentUsageCovid` here.  I think it should just be:

```
          const icuHeadroomUsed =
            row.ICUBeds.currentUsageCovid /
            (row.ICUBeds.totalCapacity - actualIcuPatientsToal);
```

The denominator is supposed to be the total ICU capacity after removing non-covid patients.

Commented on the PRD here https://docs.google.com/document/d/1Tx7_m9n5ebmxrLsIF8M8NF0KnP5vYoPPRU144zinbU0/edit?disco=AAAAJhl3-BQ

Reply:
i think we do want to subtractthe covid so it's just removing the non covid patients

Reply:
Gah! You are of course right.  I confused myself.  Sorry!